comm/src/main/scala/coop/rchain/comm/main.scala

```diff
@@ -89,7 +89,10 @@ object Main {
         }
     }
 
-    val addy = s"rnode://$name@$host:${conf.port()}"
+    val addy = p2p.NetworkAddress.parse(s"rnode://$name@$host:${conf.port()}") match {
+      case Right(node)           => node
```

Comment:
Extra whitespace. Is column-align really a thing in Scala? If so, that sucks.

Reply:
`scalafmt` does `=>`-align these things, although this looks a bit off

Reply:
You guys make the call, but we don't allow column alignment in any C/C++ because of issues like this.

Reply:
We have codified whatever the rules are in some standard `scalafmt` configuration somewhere . . . I'm ambivalent.

Reply:
yeah, its a bit of, will add scalafmt global to this project so I dont have to think about it next time

Reply:
There's an `align` setting you can put in `.scalafmt.conf` though I'm pretty sure it's off by default.  

Personally, I'm a big fan of what Nash is calling "column-alignment" (as long as you make sure to always run `scalafmt` before committing).  I think it makes the code itself more readable (i know, eye of the beholder, and all that), though certainly at the (somewhat small IMHO) expense of diff noise.  You can add `?w=1` to GitHub diff URLs to ignore whitespace changes.